roles/common/tasks/main.yml

```diff
@@ -3,12 +3,23 @@
   apt:
     name: "{{ item }}"
     state: installed
+  when: ansible_os_family == "Debian"
   with_items:
     - build-essential
     - jq
     - netcat
     - net-tools
 
+- name: Install common packages
+  yum:
```

Comment:
when: ansible_os_family == "Redhat" ??

Reply:
the other way to do all this in include_vars: "{{ ansible_os_family }" and use package: instead of apt/yum. Whatever.

Reply:
Oooops forgot the when! :) As far as using 'package:' ... at what point does it become unreadable and you have to dig in 3 different files to find the list of packages?